server/src/controllers/Events/resolver.ts

```diff
@@ -807,21 +807,39 @@ ${venueDetails}`;
       url: eventURL,
     });
 
-    const body = `When: ${event.start_at} to ${event.ends_at}<br>
-${event.venue ? `Where: ${event.venue.name}<br>` : ''}
-${event.streaming_url ? `Streaming URL: ${event.streaming_url}<br>` : ''}
-Event Details: <a href="${eventURL}">${eventURL}</a><br>
-    <br>
-    - Cancel your RSVP: <a href="${eventURL}">${eventURL}</a><br>
+    const subsequentEventEmail = `New Upcoming Event for ${
+      event.chapter.name
+    }<br />
+    <br />
+    When: ${event.start_at} to ${event.ends_at}
+    <br />
+   ${event.venue ? `Where: ${event.venue.name}.<br />` : ''}
+   ${event.streaming_url ? `Streaming URL: ${event.streaming_url}<br />` : ''}
+   <br />
+    View All of Upcoming Events for ${
+      event.chapter.name
+    }: <a href='${chapterURL}'>${event.chapter.name} chapter.</a><br />
+    RSVP or Learn More <a href="${eventURL}">${eventURL}</a>.<br />
+    -----------------------<br />
+    <br />
+    Event Details: <br />
```

Comment:
This seems to be in the wrong place (if it's needed at all), details of the event are above of it.